packages/react/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -88,6 +88,10 @@ class DropdownItem extends UIComponent<WithAsProp<DropdownItemProps>> {
     selected: PropTypes.bool,
   }
 
+  shouldComponentUpdate(props) {
+    return !_.isEqual(props, this.props)
```

Comment:
- `_.isEqual()` is the slowest possible option:
    - https://github.com/FormidableLabs/react-fast-compare
    - https://codesandbox.io/s/mystifying-volhard-wo4xb
- https://github.com/redux-form/redux-form/issues/3461#issuecomment-334425454
- https://twitter.com/dan_abramov/status/1104414272753487872
- Historical issues about `PureComponent`:
  - https://github.com/microsoft/fluent-ui-react/issues/1672#issuecomment-523910796
  - https://github.com/Semantic-Org/Semantic-UI-React/issues/1020#issuecomment-266922754